### PR TITLE
Correct hello-world README for v1

### PR DESCRIPTION
**Pull request by fcollonval**

### CODE REVIEW

File: basics/hello-world/README.md [hunk 1]
```diff
@@ -168,17 +172,17 @@ with it in the next sections.
...
 structured similarly to the extension that we are writing. This modular
 structure makes JupyterLab very adapatable_
 
 An overview of the classes and their attributes and methods can be found in the
-JupyterLab documentation. The `@JupyterLab/application` module documentation is
-[here](https://JupyterLab.github.io/JupyterLab/modules/_application_src_index_.html)
-and which links to the [JupyterLab class](https://JupyterLab.github.io/JupyterLab/classes/_application_src_index_.JupyterLab.html).
-The `JupyterLabPlugin` is a type alias [a new name] for the type `IPlugin`.
+JupyterLab documentation. The `@jupyterlab/application` module documentation is
+[here](http://jupyterlab.github.io/jupyterlab/application/index.html)
```
Comment (jtpio) [inline basics/hello-world/README.md]: Should we keep `https` here?

```suggestion
[here](https://jupyterlab.github.io/jupyterlab/application/index.html)
```

File: basics/hello-world/README.md [hunk 2]
```diff
@@ -168,17 +172,17 @@ with it in the next sections.
...
+and which links to the [JupyterFrontEnd class](http://jupyterlab.github.io/jupyterlab/application/classes/jupyterfrontend.html).
```
Comment (jtpio) [inline basics/hello-world/README.md]: Also here?

```suggestion
and which links to the [JupyterFrontEnd class](https://jupyterlab.github.io/jupyterlab/application/classes/jupyterfrontend.html).
```

**Comment by jtpio** [comment]
Thanks @fcollonval !

I left a quick suggestion. This looks good, so feel free to merge!

**Comment by fcollonval** [comment]
Thx for reviewing and catching the http.

**Pull request merged**